src/AxisManager.ts

```diff
@@ -1,9 +1,16 @@
 import Coordinate from "./Coordinate";
+import { AxesOption } from "./Axes";
 
 export interface Axis {
   [key: string]: number
 };
 
+export interface AxisOption {
```

Comment:
Sorry , my eyes had a bug.

I think `AxesOption` and `AxisOption` are the same.